rails/README.md

```diff
@@ -54,6 +54,9 @@
 - [Use blocks](/ruby/sample_2.rb#L10) when declaring date and time attributes in
   FactoryBot factories.
 - Use `touch: true` when declaring `belongs_to` relationships.
+- Use [Pundit][] when you need to restrict access to models and data.
```

Comment:
> When you are developing an application with Pundit it can be easy to forget to authorize some action. People are forgetful after all. Since Pundit encourages you to add the authorize call manually to each controller action, it's really easy to miss one.

Would it be too prescriptive to [enforce that policies and scopes are used](https://github.com/varvet/pundit?tab=readme-ov-file#ensuring-policies-and-scopes-are-used)

Reply:
I'm in favor of that! Although you should be doing one or the other. I've worked in a codebase that does both on every action and it leads to lots of weird code, unnecessary `nil`,  and skipping the before actions 😭